src/content/1.7/development/database/db.md

```diff
@@ -135,3 +135,8 @@ $result = $db->update('db_table', array(
 `_DB_PREFIX_` will be automatically prefixed to the table name.
 
 The result is boolean saying if the request was properly executed or not.
+
+### Security
+
+To avoid SQL injection, make sure to cast or sanitize your variables: use (int) for integers and pSQL() for strings.
```

Comment:
```suggestion
To prevent SQL injection, make sure variables used in queries are correctly escaped: 
 * cast with `(int)` for integers
 * use `pSQL()` for strings
```